.env.sample

```diff
@@ -14,6 +14,12 @@ REACT_APP_SUPERSET_PLANS_TABLE_SLICE=0
 REACT_APP_SUPERSET_GOALS_SLICE=0
 REACT_APP_SUPERSET_JURISDICTIONS_SLICE=0
 REACT_APP_SUPERSET_JURISDICTIONS_DATA_SLICE=0
+REACT_APP_SUPERSET_IRS_REPORTING_PLANS_SLICE=0
+REACT_APP_SUPERSET_IRS_REPORTING_JURISDICTIONS_DATA_SLICES=0,1,2,3
```

Comment:
With this we are able to load jurisdictions from as many slices as we want.  The last one in the list should be the lowest level (i.e. focus area level)

Reply:
Cool, does this assume that there is one jurisdiction level per slice? Or are we able to contain multiple levels of hierarchy per slice?

In general this makes sense if / because we have different requirements at different hierarchical levels.

Reply:
In Namibia we have all the levels in one slice.  While in Zambia we have two slices: one with all the levels apart from the lowest level; and one with just the lowest level.  This should work in both cases

Reply:
I guess an underlying assumption is that we'll usually have two slices as in Zambia's case

Reply:
> I guess an underlying assumption is that we'll usually have two slices as in Zambia's case

@moshthepitt this is a very insightful distinction to make, at the very least the multiple slices delineate between aggregated and non-aggregated data.

Another assumption this implementation has is that a single slice may contain multiple hierarchical levels and it contain a single level, and that we deduce the complete location tree the same way in both cases. (which is awesome)

Reply:
Is this explained elsewhere? If not, can we add a comment line above this constant that makes the importance of the order of the slice list clear?